Dockerfile

```diff
@@ -11,7 +11,7 @@ COPY . .
 
 RUN python -m build --wheel
 
-RUN pip wheel -w dist --no-deps --no-use-pep517 gpyfft@git+https://github.com/geggo/gpyfft@v0.8.0
+RUN pip wheel -w dist --no-deps pyvkfft
```

Comment:
pyvkfft shared library only linked to libOpenCL.so so it does not need libclfft packages.
They can be replaced with `ocl-icd-opencl-dev` and `ocl-icd-libopencl1` in build and unnamed FROM.